plugin/trino-hive/src/test/java/io/trino/plugin/hive/ion/IonPageSourceSmokeTest.java

```diff
@@ -114,6 +115,55 @@ public void testProjectedColumn()
                 1);
     }
 
+    @Test
+    public void testPageSourceWithNativeTrinoDisabled()
+            throws IOException
+    {
+        List<HiveColumnHandle> tableColumns = List.of(
+                toHiveBaseColumnHandle("foo", INTEGER, 0),
+                toHiveBaseColumnHandle("bar", VARCHAR, 1));
+        String ionText = "{ foo: 31, bar: baz } { foo: 31, bar: \"baz\" }";
+
+        TrinoFileSystemFactory fileSystemFactory = new MemoryFileSystemFactory();
+        Location location = Location.of("memory:///test.ion");
```

Comment:
Nit: Should we create a constant for location?

Reply:
Sure. Will add that since its used atleast at one other place.